db/migrate/20210623022706_create_rents.rb

```diff
@@ -0,0 +1,12 @@
+class CreateRents < ActiveRecord::Migration[6.1]
+  def change
+    create_table :rents do |t|
+      t.references :user, null: false, foreign_key: true
+      t.references :book, null: false, foreign_key: true
+      t.date :start_date
+      t.date :end_date
+
+      #t.timestamps
```

Comment:
why drop the timestamps?

Reply:
I thought that since I was already using the times for when you rent or the rent is due, the timestamps were no longer necessary

Reply:
not, the timestamp is diferent because is the life of the record, not a business logic